requirements.txt

```diff
@@ -1,10 +1,10 @@
 pyserial==3.4
 msgpack==1.0.0
-pyusb==1.0.2
-opencv-python==4.2.0.32
-requests==2.23.0
-pyzmq==19.0.0
+pyusb==1.1.0
+opencv-python==4.4.0.42
```

Comment:
@tajgr is OpenCV 4.4 working for you?

Reply:
It seems to me that there was a problem. I'll try to update my environment and give it a try.

Reply:
Test done. It is working.